app/importers/concerns/importable.rb

```diff
@@ -112,6 +116,12 @@ def parse_city_from_address(event_hash)
     end.compact.first
   end
 
+  def process_industries(entry)
+    entry[:industry] = entry[:industry] + ': ' + @naics_mapper.lookup_naics_code(entry[:industry]) if entry[:industry]
```

Comment:
You cannot really do this. If you include this concern in a class which doesn't set `@naics_mapper` then calling this method will cause an error. Better to do:

```
class A
  include Importable
  attr_accessor :naics_mapper
  def initialize
    self.naics_mapper = NaicsMapper.new
  end
end

module Importable
  extend ActiveSupport::Concern
  def process_industries(entry)
     fail 'must implement naics_mapper' unless self.class.method_defined?(:naics_mapper)
     # original implementation
  end
end
```